### DISCUSSION TITLE
support to parse string into number directly

### DISCUSSION THREAD

**Discussion by teathinker** [simdjson/simdjson#1671] [Ideas]
Hi, thanks for the excellent library.
I found some json in real world contains number as string, for example {"name": "Jason", "height", "5.9", "weight": "180"}.
Right now I think I can only parse them as string and then perform string-to-decimal conversion outside but I think it would be faster if they can be parsed within your library. Thanks.

**Comment by lemire** [comment]
@NicolasJiaxin is working on this issue. There is a pull request and a formally filed issue. I expect that we should be able to include this feature in the next release. If you'd like to help, you could provide feedback on the current pull request at https://github.com/simdjson/simdjson/pull/1667 It describes the proposed API.

**Comment by Golden-1** [comment]
can i parse decimal to string when parse a json?for example:for example {"name": "Jason", "total_count": 590990000909011119999},directly parse 590990000909011119999 to "590990000909011119999"

**Reply by lemire** [reply]
Yes, you can have raw access to it.
